src/Services/ProcessManager.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Symbiote\QueuedJobs\Services;
+
+use AsyncPHP\Doorman\Manager\ProcessManager as BaseProcessManager;
+
+/**
+ * Class ProcessManager
+ *
+ * customise shell command to allow child tasks to persist even after manager process is terminated
+ * this lets the started jobs to finish properly in case the management process terminates
+ * fore example there are no more jobs to start or queue is paused
+ *
+ * @package Symbiote\QueuedJobs\Services
+ */
+class ProcessManager extends BaseProcessManager
+{
+    /**
+     * @param string $binary
+     * @param string $worker
+     * @param string $stdout
+     * @param string $stderr
+     * @return string
+     */
+    protected function getCommand($binary, $worker, $stdout, $stderr) // phpcs:ignore SlevomatCodingStandard.TypeHints
+    {
+        return sprintf('nohup %s %s %%s %s %s & echo $!', $binary, $worker, $stdout, $stderr);
+    }
+
+    public function __destruct()
+    {
+        // Prevent background tasks from being killed when this script finishes
```

Comment:
Is there a valid reason why the default behaviour may be good? Is it worth making this behaviour a configurable value?

Reply:
The default behaviour is generally considered a more safe option and I would still want to keep it as a default. The only reason why we can bypass this here is that this module has a very good system in place which keeps track of running jobs (job locking...).

Reply:
Ok, so you're saying that it's no worth making this configurable because the queued jobs module does such a good job?
Should we do either of the following?
a) make this configurable?
b) add your rationale for why it's ok to bypass into the inline comments?

Reply:
I had a look at it in detail and I think it's worth making configurable. There might be some cases where you want this to be disabled.